src/stream.rs

```diff
@@ -346,7 +346,7 @@ impl<'a> Stream<'a> {
         Ok(())
     }
 
-    /// Consumes an XML character reference if there is one.
+    /// Consumes an XML character reference if there is one. See [try_consume_reference]
```

Comment:
I'm sorry once again, but shouldn't it bee `See [consume_reference]`?

Actually, it should be:

```rust
/// Consumes an XML character reference if there is one.
///
/// See [`consume_reference()`] for details.
///
/// [`consume_reference()`]: #method.consume_reference
```

Reply:
Ah, yes you are correct. I believe with the new intra-doc linking feature of rustdoc the explicit anchor is not necessary right?

Reply:
This library doesn't use intra-doc linking yet. It has to be converted first. It better be done as a separate pull request, if needed.